app/routers/api/remote.py

```
@@ -934,10 +927,15 @@ async def verify_transfer(
         # Calculate local hash
         from app.services.file_metadata import file_metadata_extractor
-        local_hash = await anyio.to_thread.run_sync(file_metadata_extractor.compute_sha256, final_path)
+
+        local_hash = await anyio.to_thread.run_sync(
+            file_metadata_extractor.compute_sha256, final_path
+        )
 
         if local_hash != checksum:
-            logger.error(f"Checksum mismatch! Expected {checksum}, got {local_hash}. Deleting {final_path}")
+            logger.error(
+                f"Checksum mismatch! Expected {checksum}, got {local_hash}. Deleting {final_path}"
```

Ruff (0.15.2)

[warning] Logging statement uses f-string

(G004)